src/state-summary/state-card-content.html

```diff
@@ -35,18 +35,32 @@
     'inputChanged(hass, inDialog, stateObj)',
   ],
 
-  inputChanged: function (hass, inDialog, stateObj) {
-    if (!stateObj) return;
-
+  _inputChanged: function (hass, inDialog, stateObj, stateType) {
     window.hassUtil.dynamicContentUpdater(
       this,
-      ('STATE-CARD-' +
-       window.hassUtil.stateCardType(this.hass, stateObj).toUpperCase()),
+      ('STATE-CARD-' + stateType.toUpperCase()),
       {
         hass: hass,
         stateObj: stateObj,
         inDialog: inDialog,
       });
   },
+
+  inputChanged: function (hass, inDialog, stateObj) {
+    var stateType;
+    if (!stateObj) return;
+    stateType = window.hassUtil.stateCardType(this.hass, stateObj);
+    if (stateType.indexOf('custom') !== -1) {
```

Comment:
Instead of this, I would suggest that this method becomes aware of `custom_ui_card` and not calls `stateCardType` in that case. That way you are not limited to enforcing `custom` in card names (although a 3rd party indicator is nice)